src/Rental.sol

```diff
@@ -179,8 +174,8 @@ contract Rental {
         }
 
         // If the borrower sent too much ETH, immediately refund them the extra ETH they sent 
-        if (msg.value > rentalPayment.add(collateral)) {
-            payable(msg.sender).transfer(msg.value.sub(rentalPayment.add(collateral)));
+        if (msg.value > rentalPayment + collateral) {
+            payable(msg.sender).transfer(msg.value - (rentalPayment - collateral));
```

Comment:
The final minus should be flipped here. Here are two options. 

payable(msg.sender).transfer(msg.value - (rentalPayment + collateral));

payable(msg.sender).transfer(msg.value - rentalPayment - collateral);

Which style here is up to you.

Reply:
oops good catch :pray: